junit-jupiter-params/src/test/resources/trailing-leading-spaces.csv

```diff
@@ -0,0 +1 @@
+a,b,c
```

Comment:
Please add some trailing and leading spaces here and adjust `CsvFileArgumentsProviderTests`.